### ISSUE TITLE
Code on property prediction experiments

### ISSUE THREAD

**Issue by teslacool** [DeepGraphLearning/ConfGF#2]
I failed to use Psi4 to reproduce the Property Prediction experiments and the results are very different in the order of magnitude. Can you release the code for property prediction experiments and the randomly selected 30 molecule smiles?

**Comment by shichence** [comment]
Yeah, we have the uncleaned code for the Property Prediction part. We will release that soon.
